ci-operator/jobs/openshift/gcp-pd-csi-driver-operator/openshift-gcp-pd-csi-driver-operator-main-periodics.yaml

```diff
@@ -4,23 +4,23 @@ periodics:
   cron: '@weekly'
   decorate: true
   extra_refs:
-  - base_ref: master
+  - base_ref: main
     org: openshift
     repo: gcp-pd-csi-driver-operator
   labels:
     ci-operator.openshift.io/cloud: gcp
     ci-operator.openshift.io/cloud-cluster-profile: gcp
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-gcp-pd-csi-driver-operator-master-periodic-e2e-gcp-csi-c3
+  name: periodic-ci-openshift-gcp-pd-csi-driver-operator-main-periodic-e2e-gcp-csi-c3
   reporter_config:
-    slack:
-      channel: '#forum-ocp-storage'
-      job_states_to_report:
-      - failure
-      - error
-      report_template: ':red_jenkins_circle: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
-        <{{.Status.URL}}|View logs>'
+  slack:
```

Comment:
meant the spacing here, sorry for confusion

Reply:
> meant the spacing here, sorry for confusion

thank you! let's see if that fixes the CI failures